### PR TITLE
add links

### PR DESCRIPTION
**Pull request by loanfedora**
#121

### CODE REVIEW

**Comment by etix** [comment]
For posterity, please keep the list of old users (even if the project is actually dead and the link doesn't work anymore).
Maybe make two sections, active and past users.

**Comment by loanfedora** [comment]
done @etix 

plz review

File: README.md [hunk 1]
```diff
@@ -34,21 +34,24 @@ Mirrorbits is a geographical download redirector written in [Go](https://golang.
...
+* [VideoLAN](http://www.videolan.org/) to distribute [VLC media player](http://www.videolan.org/vlc/) since [April 2014](https://blog.l0cal.com/2014/07/11/mirrorbits-is-now-on-github/)
 
 Yet some things might change before the 1.0 release. If you intend to deploy Mirrorbits in a production system it is advised to notify the author first so we can help you to make any transition as seamless as possible!
 
+_Previous projects which have used Mirrorbits:_
+* Endless OS
+* OSMC
+* Parrot OS
+* Popcorn Time
+* SuperRepo
```
Comment (etix) [inline README.md:53]:
```suggestion
* [Endless OS](https://endlessos.com/)
* [OSMC](https://osmc.tv)
* [Parrot OS](https://www.parrotsec.org)
* [Popcorn Time](https://popcorntime.io)
* [SuperRepo](https://superrepo.org)
```

Also, how do you know they're not using Mirrorbits anymore, did you contact those projects?

**Review by etix — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (loanfedora) [reply README.md:53]: dead domain
git repo or headers say another solution

**Comment by loanfedora** [comment]
thx @etix

**Pull request merged**